docs/src/pages/demos/buttons/FloatingActionButtonZoom.js

```diff
@@ -0,0 +1,130 @@
+import AppBar from 'material-ui/AppBar';
+import Tabs, { Tab } from 'material-ui/Tabs';
+import Typography from 'material-ui/Typography';
+import Zoom from 'material-ui/transitions/Zoom';
+import Button from 'material-ui/Button';
+import AddIcon from 'material-ui-icons/Add';
+import EditIcon from 'material-ui-icons/ModeEdit';
+import UpIcon from 'material-ui-icons/KeyboardArrowUp';
+import green from 'material-ui/colors/green';
+
+function TabContainer({ children, dir }) {
+  return (
+    <Typography component="div" dir={dir} style={{ padding: 8 * 3 }}>
+      {children}
+    </Typography>
+  );
+}
+
+TabContainer.propTypes = {
+  children: PropTypes.node.isRequired,
+  dir: PropTypes.string.isRequired,
+};
+
+const styles = theme => ({
+  root: {
+    backgroundColor: theme.palette.background.paper,
+    width: 500,
+    position: 'relative',
+    minHeight: 200,
+  },
+  fab: {
+    position: 'absolute',
+    bottom: theme.spacing.unit * 2,
+    right: theme.spacing.unit * 2,
+  },
+  fabGreen: {
+    background: green[500],
+  },
+});
+
+class FullWidthTabs extends React.Component {
+  state = {
+    value: 0,
+  };
+
+  handleChange = (event, value) => {
+    this.setState({ value });
+  };
+
+  handleChangeIndex = index => {
+    this.setState({ value: index });
+  };
+
+  render() {
+    const { classes, theme } = this.props;
+    const transitionDuration = {
+      enter: theme.transitions.duration.enteringScreen,
+      exit: theme.transitions.duration.leavingScreen,
+    };
+
+    const fabs = [
+      {
+        color: 'primary',
+        className: classes.fab,
+        icon: <AddIcon />,
+      },
+      {
+        color: 'accent',
+        className: classes.fab,
+        icon: <EditIcon />,
+      },
+      {
+        color: 'contrast',
+        className: classNames(classes.fab, classes.fabGreen),
+        icon: <UpIcon />,
+      },
+    ];
+
+    return (
+      <div className={classes.root}>
+        <AppBar position="static" color="default">
+          <Tabs
+            value={this.state.value}
+            onChange={this.handleChange}
+            indicatorColor="primary"
+            textColor="primary"
+            fullWidth
+          >
+            <Tab label="Item One" />
+            <Tab label="Item Two" />
+            <Tab label="Item Three" />
+          </Tabs>
+        </AppBar>
+        <SwipeableViews
+          axis={theme.direction === 'rtl' ? 'x-reverse' : 'x'}
+          index={this.state.value}
+          onChangeIndex={this.handleChangeIndex}
+        >
+          <TabContainer dir={theme.direction}>Item One</TabContainer>
+          <TabContainer dir={theme.direction}>Item Two</TabContainer>
+          <TabContainer dir={theme.direction}>Item Three</TabContainer>
+        </SwipeableViews>
+        {fabs.map((fab, index) => (
+          <Zoom
+            in={this.state.value === index}
+            timeout={transitionDuration}
+            enterDelay={transitionDuration.exit}
+            unmountOnExit
```

Comment:
I think we can remove this. It doesn't make much sense here without a matching `mountOnEnter`.

Reply:
I don't understand `mountOnEnter` is great. Why do you want to remove it?

Reply:
> without a matching mountOnEnter.

What?

Reply:
No, I was mistaken.